numba/tests/test_unicode.py

```diff
@@ -362,6 +364,22 @@ def test_concat(self, flags=no_pyobj_flags):
                                  cfunc(a, b),
                                  "'%s' + '%s'?" % (a, b))
 
+    def test_repeat(self, flags=no_pyobj_flags):
+        pyfunc = repeat_usecase
+        cfunc = njit(pyfunc)
+        for a in UNICODE_EXAMPLES + ['']:
+            for b in (-1, 0, 1, 2, 3, 4, 5, 7, 8, 15, 70):
+                self.assertEqual(pyfunc(a, b),
+                                 cfunc(a, b))
+                self.assertEqual(pyfunc(b, a),
+                                 cfunc(b, a))
+
+    def test_repeat_exception_float(self):
+        self.disable_leak_check()
+        cfunc = njit(repeat_usecase)
+        with self.assertRaises(TypingError) as raises:
+            cfunc('hi', 2.5)
+
```

Comment:
Perhaps next check `raises` actually contains the expected exception string so it's clear that the right exception has been caught, e.g., out of the with context scope do:
```
self.assertIn('Invalid use of Function(<built-in function mul>)', str(raises.exception))
```

Reply:
sure, I'll add that.